.ci/release.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+## This script runs the release given the different environment variables
+##  branch_specifier
+##  target_specifier
+##  version_override_specifier
+##
+##  NOTE: *_SECRET env variables are masked, hence if you'd like to avoid any
+##        surprises please use the suffix _SECRET for those values that contain
+##        any sensitive data. Buildkite can mask those values automatically
+
+set -e
+
+echo "--- Debug env variables"
+env | sort
+
+echo "--- Prepare vault context"
+set +x
+VAULT_ROLE_ID_SECRET=$(vault read -field=role-id secret/ci/elastic-observability-robots-playground/internal-ci-approle)
+export VAULT_ROLE_ID_SECRET
+VAULT_SECRET_ID_SECRET=$(vault read -field=secret-id secret/ci/elastic-observability-robots-playground/internal-ci-approle)
+export VAULT_SECRET_ID_SECRET
+VAULT_ADDR=$(vault read -field=vault-url secret/ci/elastic-observability-robots-playground/internal-ci-approle)
+export VAULT_ADDR
```

Comment:
This is required to access the secrets in `vault-ci` as they cannot access the `vault-prod`.

Eventually the secret location will change or likely we will use a different approach for the releases as we are in discussion with the release team and they might change the way it's done as today.